css/templates/travel/elements/_border.css

```diff
@@ -0,0 +1,24 @@
+/**
+  * Copyright 2017 The AMP Start Authors. All Rights Reserved.
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *      http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS-IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+/*
+ * border
+ *
+ */
+
+ .travel-border-gray {
```

Comment:
dont create new files for just one selector , you can add it in the main page CSS

Its okay to leave it as is if you are going to add more in future PRs

Reply:
It's used in other places (results page) :)